### DISCUSSION TITLE
Blade parsing/usage - AlpineJS/Livewire entangle

### DISCUSSION THREAD

**Discussion by mnabialek** [laravel/framework#41472] [General] [answered]
I have component with code like this:

```
<button type="button" x-data="{ on: @entangle('enabled').defer }">test</button>
```

When I have hardcoded "enabled" string, everything is working fine. The problem is that I would like to have it in variable. When using like this:

```
<button type="button" x-data="{ on: @entangle('{{ $wireModel }}').defer }">test</button>
```

I'm getting Blade parse error:

```
Unclosed '(' does not match '}'
```

Is it Blade bug or this need some fancy way to use variable here?

Disclaimer: I tried to used also:

```
<button type="button" x-data="{ on: @@entangle('{{ $wireModel }}').defer }">test</button>
```

but then getting AlpineJS console errors although page source seems fine.

**Comment by NiekNijland** [comment, marked as answer]
I have had this issue before. 

@entangle is already a blade directive, even if your IDE might not display it as such so no need for extra brackets or anything. 

You should be able to just do:

`
<button type="button" x-data="{ on: @entangle($wireModel).defer }">test</button>
`

**Reply by mnabialek** [reply]
Thanks a lot. I was pretty sure it's some JS thing and we cannot put variable here like this
